airesources/C#/Halite2/MyBot.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Halite2.hlt;
+using System.Collections.Generic;
+
+namespace Halite2
+{
+    public class MyBot
+    {
+
+        public static void Main(string[] args)
+        {
+            string name = args.Length > 0 ? args[0] : "Tamagocchi";

```

Comment:
This is a chance for you to give a default bot kit your own name, they so far have been named by bot kit authors :P